web/src/compose_validate.ts

```diff
@@ -558,7 +594,7 @@ export function validate_stream_message_mentions(opts) {
     return true;
 }
 
-export function validation_error(error_type, stream_name) {
+export function validation_error(error_type: string | undefined, stream_name: string): boolean {
```

Comment:
I don't think `error_type` can be undefined; the code here doesn't seem to expect that.